servers/cu/src/domain/client/ao-process.js

```diff
@@ -732,12 +733,15 @@ export function findLatestProcessMemoryWith ({
     }
 
     return address()
-      .chain((owner) => queryCheckpoints({
-        query: GET_AO_PROCESS_CHECKPOINTS,
-        variables: { owner, processId, limit: 50 },
-        processId,
-        before: LATEST
-      }))
+      .chain((owner) => {
+        const owners = ifElse(isEmpty, always([owner]), always(PROCESS_CHECKPOINT_TRUSTED_OWNERS))(PROCESS_CHECKPOINT_TRUSTED_OWNERS)
+        return queryCheckpoints({
+          query: GET_AO_PROCESS_CHECKPOINTS,
+          variables: { owners, processId, limit: 50 },
+          processId,
+          before: LATEST
+        })
+      })
```

Comment:
`address()` only needs to be invoked if `PROCESS_CHECKPOINT_TRUSTED_OWNERS` is not defined. See if you can refactor to to take that into account. You'll need to use `of` and/or `Resolved`